src/flatpickr.js

```diff
@@ -1801,8 +1801,9 @@ function Flatpickr(element, config) {
 				.map(dObj => self.formatDate(dObj, self.config.altFormat))
 				.join(joinChar);
 		}
-
-		triggerEvent("ValueUpdate");
+		if (triggerChange !== false) {
```

Comment:
It's only potentially bug. What if some dummy programmer reproduce this situation in application code? If you think that it's not important, i'll remove it.